### PR TITLE
🔨Refactored 🧠 Overmind Hacktober | /app/pages/common/Modals/SearchDependenciesModal/index.js : refactor to tsx

### PR DESCRIPTION
**Pull request by hetpatel33**
## What kind of change does this PR introduce?
Refactors code as a part of hacktoberfest mentioned in #2621.
@Saeris @christianalfoni @SaraVieira

## What is the current behavior?
/app/pages/common/Modals/SearchDependenciesModal/index.js was written in JS.

## What is the new behavior?
converted to tsx.

## What steps did you take to test this? This is required before we can merge.

1. Step A
2. Step B
3. Step C

## Checklist

- [ ] Documentation
- [ ] Testing 
- [ ] Ready to be merged
      
- [ ] Added myself to contributors table

### CODE REVIEW

File: packages/app/src/app/pages/common/Modals/SearchDependenciesModal/index.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useOvermind } from 'app/overmind';
 import SearchDependencies from 'app/pages/Sandbox/SearchDependencies';
 
-function SearchDependenciesModal() {
+const SearchDependenciesModal: React.FC = () => {
```
Comment (Saeris) [inline packages/app/src/app/pages/common/Modals/SearchDependenciesModal/index.tsx]:
```suggestion
export const SearchDependenciesModal: React.FC = () => {
```

File: packages/app/src/app/pages/common/Modals/SearchDependenciesModal/index.tsx [hunk 2]
```diff
@@ -13,6 +13,6 @@ function SearchDependenciesModal() {
       onConfirm={(name, version) => addNpmDependency({ name, version })}
     />
   );
-}
+};
 
 export default SearchDependenciesModal;
```
Comment (Saeris) [inline packages/app/src/app/pages/common/Modals/SearchDependenciesModal/index.tsx]:
```suggestion

```

**Comment by hetpatel33** [comment]
@Saeris 
Updated my changes as per your suggestions, please review.

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/2756/commits/322db6849770d1a29587f6b0ca2e90f0854eb018 is at https://pr2756.build.csb.dev/s/new.

**Comment by MichaelDeBoey** [comment]
Duplicate of #2747

**Pull request closed without merging**